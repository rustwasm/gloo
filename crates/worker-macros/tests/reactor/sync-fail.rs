#![no_implicit_prelude]

#[::gloo::worker::reactor::reactor]
fn Worker(_scope: ::gloo::worker::reactor::ReactorScope<(), ()>) {}

fn main() {}
