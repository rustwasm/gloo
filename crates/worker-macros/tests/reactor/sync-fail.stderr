error: reactor workers must be asynchronous
 --> tests/reactor/sync-fail.rs:4:4
  |
4 | fn Worker(_scope: ::gloo::worker::reactor::ReactorScope<(), ()>) {}
  |    ^^^^^^
